org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/ResourceUtils.java

```diff
@@ -235,14 +237,38 @@ public static boolean isContainedIn(IPath location, Collection<IPath> paths) {
 	}
 
 	/**
-	 * Expand paths starting with ~/ if necessary, or return the original path.
+	 * Expand paths starting with ~/ if necessary; replaces all the occurrences of
+	 * variables in the given string with their matching values from the environment
```

Comment:
how should those variables be defined? what format?

Reply:
Fixed.